test/http-test.js

```diff
@@ -267,8 +267,108 @@ describe('HTTP', function() {
       isvalid: true,
       address: addr.toString(node.network),
       scriptPubKey: Script.fromAddress(addr).toRaw().toString('hex'),
-      ismine: false,
-      iswatchonly: false
+      iswitness: false,
+      isscript: false
+    });
+  });
+
+  it('should not validate invalid address', async () => {
+    // Valid Mainnet P2WPKH from
+    // https://github.com/bitcoin/bips/blob/master/bip-0173.mediawiki
+    const json = await nclient.execute('validateaddress', [
+      'bc1qw508d6qejxtdg4y5r3zarvary0c5xw7kv8f3t4'
+    ]);
+
+    assert.deepStrictEqual(json, {
+      isvalid: false
+    });
+  });
+
+  it('should validate a p2wpkh address', async () => {
+    const address = 'bcrt1q8gk5z3dy7zv9ywe7synlrk58elz4hrnegvpv6m';
+    const addr = Address.fromString(address);
+    const script = Script.fromAddress(addr);
+
+    const json = await nclient.execute('validateaddress', [
+      address
+    ]);
+
+    assert.deepStrictEqual(json, {
+      isvalid: true,
+      iswitness: true,
+      address: address,
+      isscript: addr.isScripthash(),
+      scriptPubKey: script.toJSON(),
+      witness_version: addr.version,
+      witness_program: addr.hash.toString('hex')
+    });
+  });
+
+  it('should validate a p2sh address', async () => {
+    const pubkeys = [
```

Comment:
Define `pubkeys` once at the top of the `Describe` block and then provide the derivation path